backend/src/tasks/hibp.ts

```diff
@@ -40,30 +40,36 @@ async function lookupEmails(breachesDict: any, domain: Domain) {
       }
     ).json();
 
-    const AddressResults = {};
-    const BreachResults = {};
+    const addressResults = {};
+    const breachResults = {};
     const finalResults = {};
 
     const shouldCountBreach = (breach) =>
-      breach.DataClasses.indexOf('Passwords') > -1 &&
-      breach.IsVerified === true &&
-      breach.BreachDate > '2016-01-01';
+      breach.IsVerified === true && breach.BreachDate > '2016-01-01';
 
     for (const email in results) {
       const filtered = (results[email] || []).filter((e) =>
         shouldCountBreach(breachesDict[e])
       );
       if (filtered.length > 0) {
-        AddressResults[email + '@' + domain.name] = filtered;
+        addressResults[email + '@' + domain.name] = filtered;
         for (const breach of filtered) {
-          if (!(breach in BreachResults)) {
-            BreachResults[breach] = breachesDict[breach];
+          if (!(breach in breachResults)) {
+            breachResults[breach] = breachesDict[breach];
           }
         }
       }
     }
-    finalResults['Emails'] = AddressResults;
-    finalResults['Breaches'] = BreachResults;
+
+    for (const breach in breachResults) {
+      if (breachResults[breach].DataClasses.indexOf('Passwords') > -1) {
+        breachResults[breach].passwordIncluded = true;
```

Comment:
Can we define a type (using `interface` with typescript) for breachResults so we can keep track of new properties like passwordIncluded?

Reply:
@DJensen94 can you update this?